app/views/reports/regions/_header.html.erb

```diff
@@ -28,20 +28,14 @@
   %>
   <div class="d-flex fd-column fd-lg-row justify-lg-between mb-24px d-print-none">
     <div class="d-flex order-2 order-lg-1">
-      <p class="mr-12px mr-lg-24px mb-0px pb-8px fs-16px fs-lg-20px fw-bold td-none <%= params[:action] == "show" ? "c-red bb-2px bb-red" : "c-grey-dark" %>">
-        <%= link_to_unless_current "Overview", url_params.merge(action: "show"), :class => "c-grey-dark td-none" %>
-      </p>
-      <p class="mr-12px mr-lg-24px mb-0px pb-8px fs-16px fs-lg-20px fw-bold <%= params[:action] == "details" ? "c-red bb-2px bb-red" : "c-grey-dark" %>">
-        <%= link_to_unless_current "Details", url_params.merge(action: "details"), :class => "c-grey-dark td-none" %>
-      </p>
-      <p class="mr-12px mr-lg-24px mb-0px pb-8px fs-16px fs-lg-20px fw-bold <%= params[:action] == "cohort" ? "c-red bb-2px bb-red" : "c-grey-dark" %>">
-        <%= link_to_unless_current "Cohort reports", reports_region_cohort_path(@region, report_scope: params[:report_scope]), :class => "c-grey-dark td-none" %>
-      </p>
-      <% if @region.facility_region? && current_admin.feature_enabled?(:dashboard_progress_reports) %>
-        <p class="mr-12px mr-lg-24px mb-0px pb-8px fs-16px fs-lg-20px fw-bold <%= params[:action] == "cohort" ? "c-red bb-2px bb-red" : "c-grey-dark" %>">
-          <%= link_to "Progress", reports_progress_path(@region), :class => "c-grey-dark td-none" %>
-        </p>
-      <% end %>
+      <nav class="nav">
+        <%= link_to "Overview", url_params.merge(action: "show"), :class => "#{params[:action] == "show" ? "nav-link active" : "nav-link"}" %>
```

Comment:
Let's change this to

```
<%= link_to "Overview", url_params.merge(action: "show"), class: "nav-link #{active_action?("show")}" %>
```

and same for the other links below.

Reply:
Thanks. Did this.